R/Report_MetricTable.R

```diff
@@ -20,7 +20,7 @@
 #' # site-level report
 #' Report_MetricTable(
 #'   dfResults = reportingResults %>%
-#'     dplyr::filter(.data$MetricID == "kri0001") %>%
+#'     dplyr::filter(.data$MetricID == "Analysis_kri0001") %>%
```

Comment:
A lot of fixes look like this since we changed how MetricID is make in `MakeMetric()`

Reply:
Thanks for slogging through the updates.